blosc/test.py

```diff
@@ -87,19 +87,29 @@ def test_decompress_input_types(self):
         import numpy as np
```

Comment:
I think it would be better to add a new test for bytearray and keep the existing one for bytes.